nodedb.go

```diff
@@ -511,6 +511,15 @@ func (ndb *nodeDB) HasVersion(version int64) (bool, error) {
 	return ndb.db.Has(nodeKeyFormat.Key(version, []byte{1}))
 }
 
+func isReferenceToRoot(bz []byte) bool {
+	if bz[0] == nodeKeyFormat.Prefix()[0] {
+		if len(bz) == 13 {
```

Comment:
It seems the condition for a referenced node is hacky, maybe we should add a prefix to normal node values as well, so we can distinguish with prefix alone.